integr8ly/installation
# Intly 775 update che to codeready GA 1.0.0

## Additional Information
In order to do this we needed to create a non system:admin user. So we create a new user and give him clusteradmin perms to run the codeready install

## Verification Steps

#RHPDS 
Run the installer from a fresh cluster with the regular installation command
Go to the codeready ns
Use the url and login with the evals01 or admin@example.com user
Ensure you can create 2 workspaces

#OSD 
As above except you need to pass ```-e 'create_cluster_admin=false' -e 'eval_seed_users_count=1' ``` flag as we don't run as system:admin on OSD
